php-checks/src/test/resources/checks/StringLiteralDuplicatedCheck/array_keys.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+// General test for array keys - these should NOT raise issues
+"mystr";
+"mystr";
+"mystr";
```

Comment:
I don't understand this case. The comment says about array keys but I don't see any array here.

Reply:
Good catch! I added these test cases in the end related to code coverage, but it's not the right place and right associated comment.